packages/core/drizzle/0083_empty_albert_cleary.sql

```diff
@@ -0,0 +1,6 @@
+-- Custom SQL migration file, put you code below! --
+UPDATE provider_logs
+SET workspace_id = provider_api_keys.workspace_id
+FROM provider_api_keys
+WHERE provider_logs.provider_id = provider_api_keys.id
+AND provider_logs.workspace_id IS NULL;
```

Comment:
will update 80k rows which is bordering the limits of what a would do with an sql migration tbh

Reply:
Fucking scary

Reply:
Ping us if you want double check